### DISCUSSION TITLE
Kiosk Mode and Linux?

### DISCUSSION THREAD

**Discussion by brinko99** [unoplatform/uno#8747] [Q&A]
Any tips on how to use kiosk mode (no title bar and / or custom chrome) with Uno and Linux Skia.Gtk with WinUI 3? We're developing an embedded Linux app that must run as a kiosk.

I'm struggling to find such info on WinUI 3 generally that doesn't rely on Windows assigned access.

Thanks

**Comment by jeromelaban** [comment]
You can use the [`TryEnterFullScreenMode`](https://docs.microsoft.com/en-us/uwp/api/windows.ui.viewmanagement.applicationview.tryenterfullscreenmode?view=winrt-22000) API: 

```csharp
private void ToggleFullScreenModeButton_Click(object sender, RoutedEventArgs e)
{
    var view = ApplicationView.GetForCurrentView();
    if (view.IsFullScreenMode)
    {
        view.ExitFullScreenMode();
        ApplicationView.PreferredLaunchWindowingMode = ApplicationViewWindowingMode.Auto;
        // The SizeChanged event will be raised when the exit from full-screen mode is complete.
    }
    else
    {
        if (view.TryEnterFullScreenMode())
        {
            ApplicationView.PreferredLaunchWindowingMode = ApplicationViewWindowingMode.FullScreen;
            // The SizeChanged event will be raised when the entry to full-screen mode is complete.
        }
    }
}
```

**Reply by brinko99** [reply]
Excellent... this does exactly what I'm looking for.

I understand that ApplicationView is [not supported](https://docs.microsoft.com/en-us/windows/apps/desktop/modernize/desktop-to-uwp-supported-api#core-unsupported-classes) under Windows SDK. I appreciate it being supported in Uno.WinUI!

Thanks as always
